Mage.Sets/src/mage/cards/j/JaceThePerfectedMind.java

```diff
@@ -0,0 +1,101 @@
+package mage.cards.j;
+
+import mage.abilities.Ability;
+import mage.abilities.LoyaltyAbility;
+import mage.abilities.dynamicvalue.MultipliedValue;
+import mage.abilities.dynamicvalue.common.GetXLoyaltyValue;
+import mage.abilities.effects.Effect;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.MillCardsTargetEffect;
+import mage.abilities.effects.common.continuous.BoostTargetEffect;
+import mage.abilities.keyword.CompleatedAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.*;
+import mage.game.Game;
+import mage.players.Player;
+import mage.target.TargetPlayer;
+import mage.target.common.TargetCreaturePermanent;
+
+import java.util.UUID;
+
+/**
+ *
+ * @author AhmadYProjects
+ */
+public final class JaceThePerfectedMind extends CardImpl {
+
+    public JaceThePerfectedMind(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.PLANESWALKER}, "{2}{U/P}{U}");
```

Comment:
Build is failing because cost needs to be `"{2}{U}{U/P}"`

Reply:
I didnt touch the mana costs but ill change it